tokio/src/runtime/metrics/mod.rs

```diff
@@ -34,7 +35,12 @@ cfg_unstable_metrics! {
 }
 
 cfg_not_unstable_metrics! {
-    mod mock;
+    mod worker;
+    pub(crate) use worker::WorkerMetrics;
```

Comment:
Please move the modules and imports that are in both the macro blocks out above them, we don't need to gate them at all.